Source/DafnyCore/Generic/Reporting.cs

```diff
@@ -66,14 +66,14 @@ public class ConsoleErrorReporter : BatchErrorReporter {
           errorLine += "\n";
         }
 
+        Options.OutputWriter.Write(errorLine);
+
```

Comment:
The fact that this was fixed by changing the core message reporting code makes me think that some of the other messages (which already came before the snippet) aren't using this and maybe should be? Not that we need to fix that now, it's just an inconsistency we might want to resolve at some point.

Reply:
Yeah, I didn't get a chance to search for where the other messages (like normal verification failures) were bring printed from, but it's a good idea to unify them to use this logic.